vowpalwabbit/gd.cc

```diff
@@ -825,6 +825,19 @@ void save_load_online_state(
     {
       i = v.index() >> weights.stride_shift();
 
+      if (all.print_invert)  // write readable model with feature names
+      {
+        if (*v != 0.f)
+        {
+          const auto map_it = all.index_name_map.find(i);
+          if (map_it != all.index_name_map.end())
+          {
+            msg << map_it->second << ":";
+            bin_text_write_fixed(model_file, 0 /*unused*/, 0 /*unused*/, msg, true);
+          }
```

Comment:
Looks like this is missing the weight index and value as it is in the other save load function

```c++
msg << ":" <<  i << ":" << *v << "\n";
bin_text_write_fixed(model_file, 0 /*unused*/, 0 /*unused*/, msg, true);
```

Reply:
So the weight index and value are written to msg in the following code (L841 - L898).